test/ir_Argo_test.cpp

```diff
@@ -50,9 +51,11 @@ TEST(TestArgoACClass, MessageConstructon) {
   ac.setNight(true);
 
   // Don't implicitly trust this. It's just a guess.
-  uint8_t expected[kArgoStateLength] = {
-      0xAC, 0xF5, 0x00, 0x24, 0x02, 0x00, 0x00, 0x00, 0x00, 0xAC, 0xD6, 0x01};
-  EXPECT_STATE_EQ(expected, ac.getRaw(), kArgoBits);
+  auto expected = std::vector<uint8_t>({
+      0xAC, 0xF5, 0x00, 0x24, 0x02, 0x00, 0x00, 0x00, 0x00, 0xAC, 0xD6, 0x01});
+  auto actual = ac.getRaw();
+  EXPECT_THAT(std::vector<uint8_t>(actual, actual + kArgoBits / 8),
```

Comment:
This is mostly to prove out that CI links w/ GMock correctly.
The benefit in using `::ElementsAreArray` over `EXPECT_STATE_EQ` is **marginal** in this case (the only material difference is GMock's matcher would spit out all differences, and not stop on first error).

----
The wrapping into vector is there, mostly to include readability (I don't believe the extra copy for UT hurts anything).
In theory this could be written like so:
```
  EXPECT_THAT(*reinterpret_cast<uint8_t(*)[kArgoBits / 8]>(ac.getRaw()),
              ::testing::ElementsAreArray(expected));
````

Anyways, this is mostly a sample to prove integration works. As long as the i-face is a C-Style array, use of fancy matchers doesn't bring much benefit (but they start to shine once STL containers get used)